### DISCUSSION TITLE
v1.5.0

### DISCUSSION THREAD

**Discussion by AzulGarza** [Nixtla/neuralforecast#541] [Announcements]
#### New models

* [FEAT] VanillaTransformer, Autoformer in https://github.com/Nixtla/neuralforecast/pull/469
* [FEAT] StemGNN in https://github.com/Nixtla/neuralforecast/pull/456
* [FEAT] PatchTST in https://github.com/Nixtla/neuralforecast/pull/485
* [FEAT] Informer, augment_calendar_df, set seeds in fit and predict in https://github.com/Nixtla/neuralforecast/pull/463
* [FEAT] Hierarchical Forecasting Networks (HINT) in https://github.com/Nixtla/neuralforecast/pull/489

#### Misc

* [FEAT] Added MSSE class to losses.pytorch notebook in https://github.com/Nixtla/neuralforecast/pull/468
* [FEAT] Robustified Distribution Outputs in https://github.com/Nixtla/neuralforecast/pull/492
* [FEAT] Added MS availability to augment_calendar_df function in https://github.com/Nixtla/neuralforecast/pull/506
* [FEAT] Add alias argument  in https://github.com/Nixtla/neuralforecast/pull/502
* [FEAT] mean default distribution output in addition to quantiles in https://github.com/Nixtla/neuralforecast/pull/529
* [FEAT] Predict insample in https://github.com/Nixtla/neuralforecast/pull/528

### Fixes

* [FIX] Remove fixed lib versions in https://github.com/Nixtla/neuralforecast/pull/446
* [FIX] Fixed sCRPS in losses.pytorch notebook in https://github.com/Nixtla/neuralforecast/pull/462
* [FIX] Compute validation loss per epoch in https://github.com/Nixtla/neuralforecast/pull/507
* [FIX] MLP/Recurrent-based memory inference complications in https://github.com/Nixtla/neuralforecast/pull/512
* [FIX] Fix error with inference_input_size in https://github.com/Nixtla/neuralforecast/pull/536
* [FIX] Add instructions python version in https://github.com/Nixtla/neuralforecast/pull/539
* [FIX] Predict dates bug in https://github.com/Nixtla/neuralforecast/pull/540
* [FIX] Autoformer in https://github.com/Nixtla/neuralforecast/pull/523
* [FIX] Removed duplicate from model collection list in https://github.com/Nixtla/neuralforecast/pull/517

### Tutorials  and Docs

* [FEAT] Electricity Peak Detection in https://github.com/Nixtla/neuralforecast/pull/450
* [FEAT] Add End to End Walkthrough tutorial in https://github.com/Nixtla/neuralforecast/pull/472
* [DOCS] Improved HINT documentation, and broken links in https://github.com/Nixtla/neuralforecast/pull/490
* [DOCS] HINT documentation  in https://github.com/Nixtla/neuralforecast/pull/491
* [DOCS] HINT: Updated Unit Test and Example Notebooks in https://github.com/Nixtla/neuralforecast/pull/516
* [FEAT] HINT Unit Test in https://github.com/Nixtla/neuralforecast/pull/499

### New dependencies

* [FEAT] Add support for lightning>=2.0.0, and torch>=2.0.0 in https://github.com/Nixtla/neuralforecast/pull/498
* [FEAT] Allow pandas 2 in https://github.com/Nixtla/neuralforecast/pull/508

## New Contributors
* @VinishUchiha made their first contribution in https://github.com/Nixtla/neuralforecast/pull/517

**Full Changelog**: https://github.com/Nixtla/neuralforecast/compare/v1.4.0...v1.5.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Nixtla/neuralforecast/releases/tag/v1.5.0'>v1.5.0</a>.</em>
